pkg/logs/sender/sender.go

```diff
@@ -121,13 +133,15 @@ func (s *Sender) run() {
 
 		inUse := float64(time.Since(startInUse) / time.Millisecond)
 		tlmSendWaitTime.Add(inUse)
+		s.utilization.Stop()
```

Comment:
If we don't include channel wait time in the utilization, should this come before calls to `Send`?

Reply:
So the sender is a bit of a special case. 
The sender doesn't do any actual work other than dispatch payloads to destinations - which do the work of actually sending them. 

So the "work" the sender does is is really the max of all destination utilization. For example - with dual shipping, if we are sending to US1 and EU1, and we become sender constrained because the EU1 has a longer round trip, the sender's utilization will increase despite the US1 destination utilization remaining low. 

I am actually not sure how useful this will be in practice - but I included it anyway as we can always remove it if we find otherwise.